assets/components/SelectInput.jsx

```diff
@@ -30,7 +30,10 @@ const SelectInput = ({
                         <option value="">{defaultOption}</option>
                     }
                     {options.map((option) => {
-                        return <option key={option.value} value={option.value}>{gettext(option.text)}</option>;
+                        return <option 
```

Comment:
To keep this component simple, it would be best to convert the options before passing them onto this component, so this component only uses `value` and `text`

Reply:
yep